package.json

```diff
@@ -64,5 +64,6 @@
   "volta": {
     "node": "20.17.0",
     "yarn": "4.5.1"
-  }
+  },
+  "packageManager": "yarn@4.5.1"
```

Comment:
Optional, but this does make my life a bit easier, I'm having to enable/disable Volta when working in different repositories. Not sure if there's a better approach? Could be moved to a draft PR somewhere for discussion too.

- https://github.com/nodejs/corepack/issues/507#issuecomment-2625366567
- https://github.com/volta-cli/volta/issues/987

Reply:
Yes, separate PR would be good for this, as this would be something that eventually should be replicated in all our repos.